tests/python/gaia-ui-tests/gaiatest/apps/email/regions/setup.py

```diff
@@ -114,6 +114,7 @@ def type_password(self, value):
     def select_account_type(self, value):
         account_type = self.marionette.find_element(*self._account_type_locator)
         account_type.click()
+        self.marionette.switch_to_frame()
```

Comment:
Why do we need this change here? It seems unrelated to the intermittent failure.